nerdlandbot/commands/GuildData.py

```diff
@@ -73,13 +79,13 @@ def get_users_list(self, list_name: str) -> List[str]:
         """
         return self.notification_lists[list_name]["users"]
 
-    def get_emoji(self, list_name: str) -> (str,bool):
+    def get_emoji(self, list_name: str) -> (str, bool):
```

Comment:
This is invalid Python.

```suggestion
    def get_emoji(self, list_name: str) -> Tuple[str, bool]:
```

This is valid.

Reply:
Since this isn't my code I'm going to stay away from this. Maybe in the future me or someone else can fix this :D